Octokit.Tests/Clients/RepositoriesClientTests.cs

```diff
@@ -752,8 +752,7 @@ public void GetsCorrectUrl()
                 client.GetAll("owner", "name");
 
                 connection.Received()
-                    .GetAll<GitHubCommit>(Arg.Is<Uri>(u => u.ToString() == "repos/owner/name/commits"),
-                    Arg.Any<Dictionary<string, string>>());
+                    .GetAll<GitHubCommit>(Arg.Is<Uri>(u => u.ToString() == "repos/owner/name/commits"), Arg.Any<Dictionary<string, string>>(), Arg.Any<ApiOptions>());
```

Comment:
@shiftkey 
I haven't been making changes to this file.
But still the changes are visible.